_i18n/es.yml

```diff
@@ -360,17 +360,16 @@ projects:
-             <p>ASOS continues to thrive and improve and we are proud of the help we provided while there We believe that ASOS's attitude and desire for growth and change aligned well with Codurance's expertise, coaching, and support, resulting in many enriching experiences for ASOS's staff.</p>
+    subtitle: <p>El despliegue manual de muchas etapas, que requería multiples equipos, tenía un impcto negativo en la habilidad de la organización para desplegar cambios regularmente. La metodología de de Software Craftmanship fue vista como la manera de mejorar la calidad del código y proveer una plataforma hacia Continous Delivery</p>
+    about: ASOS es un negocio de moda online, vende moda de vanguardia y apoya una prospera comunidad de moda. Comenzó como un blog de moda, la marca ahora ofrece a sus clientes aplicaciones móviles y una revista además de su página web. El último año ASOS registró 13,4 millones de usuarios activos.
+    the_challenge: <p> Con más de 123 millones de visitas a su sitio al mes y con 95 millones visitas por día, ASOS se esforzaba por adaptar sus sistemas y garantizar la performance durante los períodos de máxima actividad (por ejemplo, Black Friday). Una de las claves era un proceso de despliegue manual en múltiples etapas y con múltiples equipos que requería la coordinación de un gran número de personal y pruebas manuales.</p>
+                   <p> En un esfuerzo por modernizar y mejorar sus capacidades en IT, ASOS experimentó un programa para mejorar la velocidad del cambio, la facilidad y la tasa de delivery y una mejora general del personal en esta área. Se dieron cuenta de que estaban perdiendo algunas habilidades clave, sobre todo alrededor de prácticas técnicas ágiles, y nos pidieron que ayudáramos.</p>
+    our_sc_approach: <p> Ayudamos a construir y re-arquitectar rápidamente algunos de sus sistemas principales, agregando niveles de redundancia y almacenamiento en caché que les permitió hacer frente al tráfico de pico. También construimos bibliotecas de contingencia de fallover para implementar la recuperación de desastres entre regiones e implementamos cross-region clusters de alta disponibilidad entre regiones.</p>
+                     <p> El trabajo fue en .NET ASP .NET MVC con bases de datos SQL Azure, alojado en Microsoft Azure. El sitio también empleó algunas tecnologías open source para caché de como Redis, para mejorar la disponibilidad. </p>
+                     <p>Si bien sus equipos de desarrollo ya estaban bastante establecidos, particularmente en torno a los procesos ágiles, identificamos la necesidad de que estos equipos fueran perfeccionados técnicas prácticas ágiles. Implementamos un programa de coaching de software en gran escala en sus 40 equipos de desarrollo, sobre <a href="/services">Extreme Programming (XP)</a>, TDD, principios SOLID, <a href="/services">Continuous Integration</a>, <a href="/services">Continuous Delivery</a> y <a href="/services">culture de DevOps</a> </p>
```

Comment:
`estos equipos fueran perfeccionados técnicas prácticas ágiles` should say something like `estos equipos fueran perfeccionados en técnicas prácticas ágiles`? I know the original is missing the preposition, but sounds a bit strange.

`culture de DevOps` ->  `cultura de DevOps`?